treeherder/webapp/api/performance_serializers.py

```diff
@@ -50,6 +50,17 @@ def __init__(self, *args, **kwargs):
         super(PerformanceDecimalField, self).__init__(*args, **kwargs)
 
 
+class ClassifierField(serializers.Field):
+    """Custom field for representing PerformanceAlert classifier"""
+    def to_representation(self, obj):
+        if obj is not None:
+            return obj.email
+        return obj
+
+    def to_internal_value(self, data):
+        return data
+
+
```

Comment:
I am pretty sure that you should be able to implement this more simply with a slug related field: http://www.django-rest-framework.org/api-guide/relations/#slugrelatedfield

Please give it a try!